examples/api/lib/cupertino/checkbox/cupertino_checkbox.0.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+/// Flutter code sample for [CupertinoCheckbox].
+
+void main() => runApp(const CupertinoCheckboxApp());
+
+class CupertinoCheckboxApp extends StatelessWidget {
+  const CupertinoCheckboxApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const CupertinoApp(
+      theme: CupertinoThemeData(brightness: Brightness.light),
+      home: CupertinoPageScaffold(
+        navigationBar: CupertinoNavigationBar(
+          middle: Text('CupertinoCheckbox Example'),
+        ),
+        child: SafeArea(
+          child: CupertinoCheckboxExample(),
+        ),
+      ),
+    );
+  }
+}
+
+class CupertinoCheckboxExample extends StatefulWidget {
+  const CupertinoCheckboxExample({super.key});
+
+  @override
+  State<CupertinoCheckboxExample> createState() => _CupertinoCheckboxExampleState();
+}
+
+class _CupertinoCheckboxExampleState extends State<CupertinoCheckboxExample> {
+  bool? isChecked = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoCheckbox(
+      checkColor: CupertinoColors.white,
+      tristate: true,
```

Comment:
Maybe add a comment above the `tristate` property explaining what it does, and suggesting they comment it out to see how the behaviors change.

Reply:
👍